armedilla_alliyah_samantha/country.js

```diff
@@ -0,0 +1,62 @@
+const countryInput = document.getElementById('country-input');
+const searchBtn = document.getElementById('search-btn');
+const countryDetails = document.getElementById('country-details');
+const regionCountries = document.getElementById('region-countries');
```

Comment:
If you're done changing IDs in your country.html, do not forget to change also your ID naming in country.js, so that it will get the appropriate id. For example:
❌ const countryInput = document.getElementById('country-input');
✅ const countryInput = document.getElementById('country_input');

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5